data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,323 @@
+				action
+					set "incipias: helped the derelict"
+		personality
+			confusion 50
+			derelict uninterested staying
+	on enter "Hui'uc"
+		conversation
+			`On entering the system you find an alien spaceship disabled and badly damaged. There does not appear to be any other ships or inhabited planets in the system.`
+
+mission "Incipias: Help the stranded Part 1"
+	landing
+	source
+		government "Quarg (Incipias)"
+	destination "Pon'tes"
+	to offer
+		has "incipias: helped the derelict"
+	on offer
+		conversation
+			`Right after you arrived on the planet you go looking for a Quarg official to tell about the stranded ship. Luckily it does not take long before you find one.`
+			`	"Human, you seem to be in quite a haste. What could be the so pressing matter?" it asks.`
+			choice
+				`	(Explain the situation.)`
+					goto explanation
+				`	"I need advice regarding the species in this region."`
+			`	It looks at you curiously, "So you met them? I am surprised that you were able to communicate. It took a very long time even for us to decipher their way of communication."`
+			`	You quickly explain what happened. The Quarg seems very relieved about the fact that you were not able to talk to them.`
+			label explanation
+			`	"You ran into a most peculiar situation, this species should not be able to cross the borders of their system yet..." It tilts it head for a moment considering what you just said. "I would love to help them, but we have no ships that can land on gas giants and it is too early for us to reveal ourselves to them, I fear there is nothing we can do." It shakes its head.`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"How can you make such a cold-hearted decision?"`
+			`	"This might be difficult for you to understand, but I have to place the greater good over the sole life. If we show ourselves to them now by the shock of such and advanced species existing chaos will be created."`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"You can deal with the consequences later. Help them now!"`
+			`	"Insisting you sure are human, understand I can your point but intervention by us out of question it is. But a proposal for you I have, to you, a solo captain the means to help I will grant, of course some conditions there are."`
+			choice
+				`	"Sure, let me do the work."`
+				`	"If you are too weak..."`
+			`	Something like a laugh comes from the Quarg, before continuing.`
+			label initiative
+			`	"Your initiative most welcome is human, but not the tools for such a mission you have. To my knowledge no vessels capable of landing on gas giants ever developed by your kind. And as said you, not capable of communicating with them you are."`
+			`	"But help you I can at one point, communication that is. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. To sell you one of their ships ask to them, easy it won't be."`
```

Comment:
```suggestion
			`	"But on this last point I am able to assist you. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. Request that they sell you one of their ships, so that you are able to land on their planets."`
```